### PR TITLE
R11

### PR DESCRIPTION
**Pull request by ninggao**
Fixes #

## Proposed Changes

  - 
  -
  -

### CODE REVIEW

File: Orchestrator/docs/BFOrchestratorReport.md [hunk 1]
```diff
@@ -1,109 +1,146 @@
...
-In this section, the report lists the utterances tagged with more than one labels.
-Sometimes some dataset might contain utterances tagged with the same labels multiple times.
+This section reports on utterances with duplicate or multiple labels. A duplicate utterance is detected when it is present more than once. Thus, the report lists the utterances tagged with more than one labels. Sometimes some dataset might contain utterances tagged with the same labels multiple times.
+
 The report also lists the redundancy.
 
 - Multi-label utterances and their labels
 - Duplicate utterance and label pairs
 
+### How to use this section
```
Comment (tsuwandy) [inline Orchestrator/docs/BFOrchestratorReport.md:57]: Do we need this TBD?  Just remove and add it later?

File: Orchestrator/docs/BFOrchestratorReport.md [hunk 2]
```diff
@@ -1,109 +1,146 @@
...
+This section reports on utterances with incorrect predictions. An a misclassified predication is one in which an Orchestrator model falsely predicts its intent label. Usually the label with the highest prediction score is chosen as the predicted label, but
 it can be different from the ground-truth label for the utterance.
 
 Similar to the last section, the report also lists the prediction and ground-truth labels with
 their prediction scores and closest examples.
 
+
+
+### How to use this section
+
```
Comment (tsuwandy) [inline Orchestrator/docs/BFOrchestratorReport.md:107]: Just remove the TBD?  Can always add whatever we need to add later?

File: Orchestrator/docs/BFOrchestratorUsage.md [hunk 3]
```diff
@@ -18,15 +18,15 @@ The mainstream bot language recognition development cycle with Orchestrator is g
...
+### 1. Create Intent label / utterance examples file in [.lu format][2]
 
 If you are developing a new language model, simply refer to [Language understanding][8] documentation to author label files. Depending on how you plan to design your bot, you may use a single label file corresponding to a single snapshot file for the entire bot or multiple pairs, each for every adaptive dialog where you plan to use Orchestrator as a recognizer. 
 
 In case of migration from legacy dispatch, you may need to retrieve your LUIS application language model using the ```bf luis:version:export``` command to .lu file or QnA Maker knowledgebase using ```bf qnamaker:kb:export```  to .qna format. 
 
 **TBD**: See sample (or example line) here...
 
-### 2. Download Orchestrator base model
+### 2. Download Natural Language Representation ([NLR][4]) base model
```
Comment (tsuwandy) [inline Orchestrator/docs/BFOrchestratorUsage.md:29]: We are no longer calling it NLR, should keep base model

**Review by tsuwandy — APPROVED** [review]

File: Orchestrator/docs/BFOrchestratorReport.md (hunk 1 above)
Comment (ninggao) [reply Orchestrator/docs/BFOrchestratorReport.md:57]: This is the placeholder for Hung-chih. Once I'm done with these files, Hung-chih will work on those.

File: Orchestrator/docs/BFOrchestratorReport.md (hunk 2 above)
Comment (ninggao) [reply Orchestrator/docs/BFOrchestratorReport.md:107]: Hung-chih was planning to work on it once I'm done here. This is his placeholder.

File: Orchestrator/docs/BFOrchestratorUsage.md (hunk 3 above)
Comment (ninggao) [reply Orchestrator/docs/BFOrchestratorUsage.md]: Changed to base model.

**Pull request merged**